moveit_ros/moveit_servo/config/servo_parameters.yaml

```
@@ -191,18 +210,60 @@ servo:
                   this should be set to false"
   }
 
-  halt_all_joints_in_joint_mode: {
+############################### SMOOTHING PLUGIN ###############################
+
+  use_smoothing: {
     type: bool,
+    read_only: true,
     default_value: true,
-    description: "Halt all joints in joint mode"
+    description: "Enables the use of smoothing plugins for joint trajectory smoothing"
   }
 
-  halt_all_joints_in_cartesian_mode: {
+  smoothing_filter_plugin_name: {
+    type: string,
+    read_only: true,
+    default_value: "online_signal_smoothing::ButterworthFilterPlugin",
+    description: "The name of the smoothing plugin to be used"
+  }
+
+############################# COLLISION MONITOR ################################
+
+  check_collisions: {
     type: bool,
     default_value: true,
-    description: "Halt all joints in cartesian mode"
+    description: "If true, servo will check for collision using the planning scene monitor."
+  }
+
+  self_collision_proximity_threshold: {
+    type: double,
```

If you look at collision_monitor.cpp, this is involved in variable initialization. Can't be changed at runtime.

```suggestion
    type: double,
    read_only: true,
```